eslint-disable-rule-flycheck.el

```diff
@@ -60,11 +60,23 @@ but useless, to use even when not using flycheck."
     (when-let* ((error (ignore-errors (or
                                        (get-char-property (point) 'flycheck-error)
                                        (flycheck-next-error-function 1 nil))))
-                (rule-name (flycheck-error-id error)))
+                (rule-name (and
```

Comment:
even if both works the same here, I think `when` makes more sense than `and` in this case because we care about the return value of `flycheck-error-id`, not only the boolean value of `and`.